python/ray/data/dataset.py

```diff
@@ -2380,6 +2380,27 @@ def to_tf(
         if isinstance(output_signature, list):
             output_signature = tuple(output_signature)
 
+        def get_df_values(df: "pandas.DataFrame") -> np.ndarray:
+            # TODO(Clark): Support unsqueezing column dimension API, simialr to
+            # to_torch().
+            try:
```

Comment:
Could you add a comment explaining what this try-except is checking for?

Reply:
There's a comment on the need for this exception handling in the except clause below.

Reply:
Oh oops, that sounds good!